### PR TITLE
Markdownlint fixes

### PR DESCRIPTION
**Pull request by mdo**
Addresses some Markdown linting found from testing out `markdownlint` in #29919.

### CODE REVIEW

File: site/content/docs/5.0/components/card.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Cards are built with as little markup and styles as possible, but still manage t
 Below is an example of a basic card with mixed content and a fixed width. Cards have no fixed width to start, so they'll naturally fill the full width of its parent element. This is easily customized with our various [sizing options](#sizing).
 
 {{< example >}}
-<div class="card" style="width: 18rem;">
+<div class="card col-md-6 col-lg-5">
```
Comment (XhmikosR) [inline site/content/docs/5.0/components/card.md:20]: These seem unrelated to the patch?

**Comment by XhmikosR** [comment]
@mdo did you use my branch https://github.com/twbs/bootstrap/tree/master-xmr-markdownlint? Because there are false positives Markdownlint cannot handle hence why I haven't made a PR to actually add Markdownlint.

File: site/content/docs/5.0/components/card.md (hunk 1 above)
Comment (mdo) [reply site/content/docs/5.0/components/card.md:20]: Oh damn, yes, will fix.

**Comment by mdo** [comment]
I didn't use your branch—I use markdownlint at GitHub and just ran the CLI against our docs to clear out the easy wins. There are tons of false positives due to our docs tags and what not.

File: README.md [hunk 2]
```diff
@@ -193,16 +193,16 @@ Thanks to [BrowserStack](https://www.browserstack.com/) for providing the infras
...
-[![](https://opencollective.com/bootstrap/sponsor/2/avatar.svg)](https://opencollective.com/bootstrap/sponsor/2/website)
-[![](https://opencollective.com/bootstrap/sponsor/3/avatar.svg)](https://opencollective.com/bootstrap/sponsor/3/website)
-[![](https://opencollective.com/bootstrap/sponsor/4/avatar.svg)](https://opencollective.com/bootstrap/sponsor/4/website)
-[![](https://opencollective.com/bootstrap/sponsor/5/avatar.svg)](https://opencollective.com/bootstrap/sponsor/5/website)
-[![](https://opencollective.com/bootstrap/sponsor/6/avatar.svg)](https://opencollective.com/bootstrap/sponsor/6/website)
-[![](https://opencollective.com/bootstrap/sponsor/7/avatar.svg)](https://opencollective.com/bootstrap/sponsor/7/website)
-[![](https://opencollective.com/bootstrap/sponsor/8/avatar.svg)](https://opencollective.com/bootstrap/sponsor/8/website)
-[![](https://opencollective.com/bootstrap/sponsor/9/avatar.svg)](https://opencollective.com/bootstrap/sponsor/9/website)
+[![OC sponsor](https://opencollective.com/bootstrap/sponsor/0/avatar.svg)](https://opencollective.com/bootstrap/sponsor/0/website)
```
Comment (XhmikosR) [inline README.md:196]: I'm not sure if "OC Sponsor" is better than nothing. @patrickhlauke

**Comment by XhmikosR** [comment]
> There are tons of false positives due to our docs tags and what not.

I have disabled some rules in my branch https://github.com/twbs/bootstrap/runs/589127421#step:9:235

**Comment by mdo** [comment]
To clarify, this isn't meant to add a script or Action anywhere—it's just cleaning up some Markdown :). I'll defer to another PR for implementing it and running it as a check for us everywhere.

**Comment by XhmikosR** [comment]
>To clarify, this isn't meant to add a script or Action anywhere

I know, just trying to get the best of the two branches :)

So, you prefer a `-` for ul style?

That being said, let's wait to hear about the `alt` change from @patrickhlauke. Personally I just ignored these specific errors in README.md in my branch and allowed them everywhere else.

File: README.md (hunk 2 above)
Comment (XhmikosR) [reply README.md:196]: Let's revert this too @mdo since the patch is only about the dashes for `ul`s.

I will adapt my branch to use dashes too later

**Comment by XhmikosR** [comment]
@patrickhlauke friendly ping :)

File: README.md (hunk 2 above)
Comment (patrickhlauke) [reply README.md:196]: so i take it there's no way for us to somehow get that pulled in via third party or something and have it provided with more accurate alt etc? (like a validation badge or similar that actually gets generated separately and dynamically pulled in)

if not, then...yeah, better than nothing i guess

Comment (XhmikosR) [reply README.md:196]: Unfortunately, I'm not aware of a way. But perhaps we should use `OC sponsor 1` etc?

Comment (patrickhlauke) [reply README.md:196]: i guess you could. that would still not provide the equivalent of what sponsor it is, but i guess it would at least indicate it's not always the same image. least worst fix, probably

**Review by XhmikosR — APPROVED** [review]
